eng/pipelines/migration.yml

```diff
@@ -0,0 +1,118 @@
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/recordings/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+# The input parameter format is like:
+# - MigrateFrom: azsdk-pool-mms-win-2019-general
+#   MigrateTo: azsdk-pool-mms-win-2022-general
+# - MigrateFrom: windows-2019
+#   MigrateTo: windows-2022
+# - MigrateFrom: windows2019
+#   MigrateTo: windows2022
+# - MigrateFrom: MMS2019
+#   MigrateTo: MMS2022
+- name: MigrationMap
+  type: object
+  default: []
+- name: ShortForMigrateFrom
+  type: string
+  default: ''
+- name: ShortForMigrateTo
+  type: string
+  default: ''
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+jobs:
+  - job: Migration
+    pool:
+      name: azsdk-pool-mms-ubuntu-2004-general
+    variables:
+      - template: ./templates/variables/globals.yml
+    steps:
+      - ${{ each repo in parameters.Repos }}:
+        - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
+            SkipCheckoutNone: true
+            Repositories:
+              - Name: ${{ repo.RepoOwner }}/${{ repo.RepoName }}
+                WorkingDirectory: $(Build.SourcesDirectory)/docs/${{ repo.RepoName }}
+            Paths:
+              - /*
+        - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+        - pwsh: |
+            $migrationMap = '${{ convertToJson(parameters.MigrationMap) }}'
+            $(Build.SourcesDirectory)/eng/scripts/Repo-File-Content-Replacements.ps1 `
+              -ExcludePathsRegex '${{ repo.ExcludePaths }}' `
+              -IncludePathsRegex '${{ repo.IncludePaths }}' `
+              -MigrationMap $migrationMap
+          displayName: Migration for "${{ repo.RepoName }}"
+          workingDirectory: $(Build.SourcesDirectory)/docs/${{ repo.RepoName }}
+          continueOnError: true
+        - ${{ if ne(parameters.WhatIfPreference, 'true')}}:
+          - pwsh: |
+              $teamreviewers = "azure-sdk-eng"
+              if ($env:TEAMREVIEWERS) {
+                $teamreviewers = $env:TEAMREVIEWERS
+              }
+              Write-Host "##vso[task.setvariable variable=TeamReviewers]$teamreviewers"
+            displayName: Set azure-sdk-eng as reviewers
+          - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
+            parameters:
+              BaseBranchName: $(DefaultBranch)
+              PRBranchName: migration
+              CommitMsg: "Migrate from ${{parameters.ShortForMigrateFrom}} to ${{parameters.ShortForMigrateTo}}"
+              RepoOwner: ${{ repo.RepoOwner }}
+              RepoName: ${{ repo.RepoName }}
+              PRTitle: "Migrate from ${{parameters.ShortForMigrateFrom}} to ${{parameters.ShortForMigrateTo}}"
+              WorkingDirectory: $(Build.SourcesDirectory)/docs/${{ repo.RepoName }}
+              # Var name here should the same as the above step.
+              GHTeamReviewersVariable: TeamReviewers
```

Comment:
What the team reviewer added after your tests? As I don't see it in any of the test PRs created.

Reply:
I did not make it work. :( I forgot to test this. Will spend time to fix this.

Reply:
Synced offline. We will use user reviewers here. People can add github alias directly to the parameter `RequestBy`